Bug(?): ng serve --aot on develop (commit 1c4f00a7d...)

### Do you want to request a *feature* or report a *bug*? bug
#### Bug Report
{{{
$ ng --version
... banner ...
angular-cli: 1.0.0-beta.25.5
node: 6.9.4
os: linux x64
@angular/common: 2.4.4
@angular/compiler: 2.4.4
@angular/core: 2.4.4
@angular/forms: 2.4.4
@angular/http: 2.4.4
@angular/material: 2.0.0-beta.1
@angular/platform-browser: 2.4.4
@angular/platform-browser-dynamic: 2.4.4
@angular/platform-server: 2.4.4
@angular/router: 3.4.4
@angular/compiler-cli: 2.4.4
$ tsc --version
Version 2.0.10

$ ng serve --aot
** NG Live Development Server is running on http://localhost:4200. **
Hash: 13802425a5088a993bdf                                                               
Time: 24836ms
chunk    {0} styles.bundle.js, styles.bundle.map (styles) 483 kB {4} [initial] [rendered]
chunk    {1} scripts.bundle.js, scripts.bundle.map (scripts) 97.4 kB {4} [initial] [rendered]
chunk    {2} main.bundle.js, main.bundle.map (main) 1.33 kB {3} [initial] [rendered]
chunk    {3} vendor.bundle.js, vendor.bundle.map (vendor) 1.91 MB [initial] [rendered]
chunk    {4} inline.bundle.js, inline.bundle.map (inline) 0 bytes [entry] [rendered]

ERROR in Cannot determine the module for class FakeComponent in /home/incze/ws/learn/angular/covalent/src/platform/core/common/services/router.service.spec.ts!

ERROR in ./src/main.ts
Module not found: Error: Can't resolve './$$_gendir/app/app.module.ngfactory' in '/home/incze/ws/learn/angular/covalent/src'
 @ ./src/main.ts 4:0-74
 @ multi main

ERROR in ./~/@angular/core/src/linker/system_js_ng_module_factory_loader.js
Module not found: Error: Can't resolve '/home/incze/ws/learn/angular/covalent/src/$$_gendir' in '/home/incze/ws/learn/angular/covalent/node_modules/@angular/core/src/linker'
 @ ./~/@angular/core/src/linker/system_js_ng_module_factory_loader.js 71:15-36 87:15-102
 @ ./~/@angular/core/src/linker.js
 @ ./~/@angular/core/src/core.js
 @ ./~/@angular/core/index.js
 @ ./src/main.ts
 @ multi main
webpack: bundle is now VALID.
}}}

--- a/src/tsconfig.json
+++ b/src/tsconfig.json
@@ -25,5 +25,8 @@
     "noUnusedLocals": false,
     "allowUnreachableCode": false,
     "pretty": true
-  }
+  },
+  "exclude": [
+    "**/*.spec.*"
+  ]
 }

--- a/src/tsconfig.json
+++ b/src/tsconfig.json
@@ -27,6 +27,6 @@
     "pretty": true
   },
   "exclude": [
-    "**/*.spec.ts"
+    "**/*.spec.*"
   ]
 }

--- a/src/tsconfig.json
+++ b/src/tsconfig.json
@@ -27,6 +27,6 @@
     "pretty": true
   },
   "exclude": [
-    "**/*.spec.*"
+    "**/*.spec.ts"
   ]
 }

